packages/sdks/web-component/src/lib/types.ts

```diff
@@ -99,11 +125,21 @@ export type StepState = {
   openInNewTabUrl?: string;
 } & OIDCOptions;
 
+export type CustomScreenState = Omit<
+  ScreenState,
+  'cssVars' | 'componentsConfig' | 'inputs'
```

Comment:
should the omit values match also client scripts, errorText, etc?

![image](https://github.com/user-attachments/assets/b321c594-e237-4d79-80e9-f9c294e4b013)

Reply:
Yes, the errors keys are under an error object
The client scripts are running in the SDK, the descoper does not need to run it as part of his screen
If you think there are more keys that should be added/removed please tell